plugins/utils/harpoon.nix

```diff
@@ -144,9 +144,16 @@ in
         Menu window height
       '';
 
-      borderChars = helpers.defaultNullOpts.mkNullable (
-        with types; listOf str
-      ) ''["─" "│" "─" "│" "╭" "╮" "╯" "╰"]'' "Border characters";
+      borderChars = helpers.defaultNullOpts.mkNullable (with types; listOf str) [
```

Comment:
```suggestion
      borderChars = helpers.defaultNullOpts.mkListOf types.str [
```
?

Reply:
Good catch.

Eventually I lost motivation for changing _all_ cases of `mkNullable listOf` to `mkListOf`, but I can give this another pass if needed.